deploy/helm/charts/templates/cspc-operator.yaml

```
@@ -45,17 +45,21 @@ spec:
               valueFrom:
                 fieldRef:
                   fieldPath: metadata.name
+{{- if .Values.cspcOperator.baseDir }}
             # OPENEBS_IO_BASE_DIR is used to configure base directory for openebs on host path.
             # Where OpenEBS can store required files. Default base path will be /var/openebs
-            # - name: OPENEBS_IO_BASE_DIR
-            #   value: "/var/openebs"
+            - name: OPENEBS_IO_BASE_DIR
+              value: {{ .Values.cspcOperator.baseDir | quote }}
+{{- end }}
+{{- if .Values.cspcOperator.SparseDir }}
```

Simillarly, in all other places
```suggestion
{{- if .Values.cspcOperator.sparseDir }}
```